dashboard/config/routes.rb

```diff
@@ -617,6 +617,13 @@
             patch :bulk_update_owners
           end
         end
+        resource :account_linking, only: [], controller: 'account_linking' do
+          collection do
+            get :landing
+            get :link_existing_account
+            post :link_email
+          end
+        end
```

Comment:
Since `account_linking` is not technically a resource, it would be more appropriate to group the actions within the `account_linking` namespace:

```bash
                                      Prefix Verb URI Pattern                                             Controller#Action
              lti_v1_account_linking_landing GET  /lti/v1/account_linking/landing(.:format)               lti/v1/account_linking#landing
lti_v1_account_linking_link_existing_account GET  /lti/v1/account_linking/link_existing_account(.:format) lti/v1/account_linking#link_existing_account
           lti_v1_account_linking_link_email POST /lti/v1/account_linking/link_email(.:format)            lti/v1/account_linking#link_email
```

```suggestion
        namespace :account_linking do
          get :landing
          get :link_existing_account
          post :link_email
        end
```